packages/components/src/FormField/FormField.tsx

```diff
@@ -66,7 +67,7 @@ export function FormField(props: FormFieldProps) {
 
```

Comment:
Should we proactively set the `{shouldValidate:true}` in the [useEffect here](https://github.com/GetJobber/atlantis/pull/1176/files#diff-7664eb8fb7178dac2dae4e5ccb337b655a800ab7dade9a2ec9501c0d428c78f7R64) as well??

Reply:
I think @darryltec or @MichaelParadis would have a better idea on it than myself about it

Reply:
I think so. Mainly because the revalidate mode will be `onChange` so we should trigger validations since `setValue` doesn't by default